src/DB_Command.php

```diff
@@ -320,6 +332,16 @@ public function export( $args, $assoc_args ) {
 			}
 		}
 
+		if ( isset( $assoc_args['skip-tables'] ) ) {
```

Comment:
You should use `WP_CLI\Utils\get_flag_value()` here, to ensure `--no-exclude_tables` could be passed to negate any standard arguments.